accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
 different sets of document types available in Chile. By default, all the sales journals created will
 use documents. 
   
-.. image:: media/Chile10.png
```

Comment:
Is this file still used somewhere?

Reply:
No, it has been removed in the last commt